CONTRIBUTING.md

```diff
@@ -12,6 +12,7 @@ Please make sure the following company details are provided:
 
 Some additional things to keep in mind:
 * The company must allow working in/from Portugal;
+* Listed offices must allow for development/tech jobs (e.g. a tech company with HR offices in Portugal should not be listed);
```

Comment:
I would phrase the phrase within parenthesis more clearly regarding a location

 `e.g. a location with only HR functions should not be listed`

Reply:
Makes sense!